RELEASE.md

```diff
@@ -18,21 +18,27 @@ All the Python packaging instructions in the `pyproject.toml` file to wrap your
 > [!IMPORTANT]
 > There is no need to update the version in `pyproject.toml` as it is automatically updated by `hatch` when generating the package.
 
-3. To create a Python source package (`.tar.gz`) and the binary package (`.whl`) in the `dist/` directory, do:
+3. Optional - if there is a newer release of `conda-store-ui`, update the `conda-store-ui` dependency in the package.json file.
+
+   ```bash
+   yarn upgrade @conda-store/conda-store-ui@<version>
+   ````
```

Comment:
Thanks for a quick merge over on https://github.com/conda-incubator/jupyterlab-conda-store/pull/44. Now the thing needed to please the linter is fixing the extra backtick here:

```suggestion
   ```
```

Reply:
Note: my suggestion does not render properly because of some GitHub UI limitation. I do not suggest to deleted it.

Reply:
was already fixing this lol